src/aws-manager.js

```diff
@@ -285,7 +260,12 @@ class AwsManager {
             let workerType = this.parseKeyPairName(instance.KeyName).workerType;
             instance.Region = region;
             instance.WorkerType = workerType;
-            apiState.instances.push(instance);
+            // apiState for instance state name of 'running', 'pending'
+            if (_.includes(['running', 'pending'], instance.State.Name)) {
+              apiState.instances.push(instance);
+            } else if (_.includes(['shutting-down', 'terminated', 'stopping'], instance.State.Name)) {
+              deadState.instances.push(instance);
+            }
             ignoreSR.push(instance.SpotInstanceRequestId);
```

Comment:
`ignoreSR` are spot request ids which are for an instance.  If a SRID is associated with an instance, we know that the spot request it's for has be filled.  If an SR has been filled but the describeSpotInstance requests call doesn't reflect that due to eventual consistency, we now know that we can ignore it to avoid double counting it.

The case I'm concenred with for ignoreSR is that the describe calls are out of sync, so we're double counting some spot requests.